pom.xml

```diff
@@ -50,14 +50,37 @@
             <plugin>
                 <groupId>org.apache.felix</groupId>
                 <artifactId>maven-bundle-plugin</artifactId>
+                <version>5.1.1</version>
```

Comment:
Migrating to bnd is required with newest parent: https://github.com/apache/sling-parent/blob/133c73392e9575cb10ced19ce29e14e7ec61c790/sling-bundle-parent/pom.xml#L168

Reply:
There are 2 parent poms: sling 39 and sling-bundle-parent 39. This project has used sling and I just updated that to the latest of that one. sling 39 does not require migrating to bnd, it's the other one that does.

If you veto this PR because you think it must be migrated to sling-bundle-parent please let me know, otherwise I suggest that we merge this PR and the migration to the bnd-maven-plugin and sling-bundle-parent can be done separately.

Reply:
I don’t require update to bnd-m-p. But please don’t upgrade to parent above 34 as that does no longer properly support m-bundle-plugin

Reply:
Ok - I downgraded to 34, so I guess this part can be marked as resolved?